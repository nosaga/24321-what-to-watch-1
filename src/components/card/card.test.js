import React from "react";
import renderer from "react-test-renderer";
import Card from "./card.jsx";

const mock = {
  src: ``,
  title: ``,
};

it(`Card renders correctly`, () => {
  const {
    src,
    title,
  } = mock;
  const tree = renderer
    .create(<Card
      src={src}
      title={title}
    />)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
