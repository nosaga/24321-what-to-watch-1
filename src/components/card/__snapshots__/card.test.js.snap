// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card renders correctly 1`] = `
<article
  className="small-movie-card catalog__movies-card"
>
  <button
    className="small-movie-card__play-btn"
    type="button"
  >
    Play
  </button>
  <div
    className="small-movie-card__image"
  >
    <img
      alt=""
      height="175"
      src="img/fantastic-beasts-the-crimes-of-grindelwald.jpg"
      width="280"
    />
  </div>
  <h3
    className="small-movie-card__title"
  >
    <a
      className="small-movie-card__link"
    >
      
    </a>
  </h3>
</article>
`;
